WordPress/src/test/java/org/wordpress/android/ui/posts/mediauploadcompletionprocessors/TestContent.kt

```diff
@@ -357,6 +360,92 @@ object TestContent {
   </ul>
 </figure>
 <!-- /wp:gallery -->
+"""
+
+    const val oldCoverBlock = """<!-- wp:cover {"url":"$localImageUrl","id":$localMediaId,"someOtherAttribute":5} -->
+<div class="wp-block-cover has-background-dim" style="background-image:url($localImageUrl)">
+  <div class="wp-block-cover__inner-container">
+    <!-- wp:paragraph {"align":"center","placeholder":"Write title…"} -->
+    <p class="has-text-align-center"></p>
+    <!-- /wp:paragraph -->
+  </div>
+</div>
+<!-- /wp:cover -->
+"""
+
+    const val newCoverBlock = """<!-- wp:cover {"url":"$remoteImageUrl","id":$remoteMediaId,"someOtherAttribute":5} -->
+<div class="wp-block-cover has-background-dim" style="background-image:url($remoteImageUrl)">
+  <div class="wp-block-cover__inner-container">
+    <!-- wp:paragraph {"align":"center","placeholder":"Write title…"} -->
+    <p class="has-text-align-center"></p>
+    <!-- /wp:paragraph -->
+  </div>
+</div>
+<!-- /wp:cover -->
+"""
+    const val oldCoverBlockWithNestedImageBlock = """<!-- wp:cover {"url":"$localImageUrl","id":$localMediaId} -->
```

Comment:
Hey, @mkevins Just want to confirm this flow. This is this data supposed to represent a new cover block and a new image block within it or is it supposed to be an existing cover block with a new image block? If it is supposed to be new at both layers in practice these will have different IDs correct? 

Not a big deal but want to make sure I understand correctly for the way I approach it on iOS